Assets/GameAssets/Player/PlayerScripts/PlayerHealth.cs

```diff
@@ -37,12 +44,41 @@ private void OnDestroy()
     /// Performs the base functionality then calls player related event
     /// </summary>
     /// <param name="heal"> The amount of healing received </param>
-
     public override void IncreaseHealth(float heal)
     {
         base.IncreaseHealth(heal);
 
         PlayerManager.Instance.InvokePlayerHealthChangeEvent(GetHealthPercent(), _currentHealth);
+        PlayHeartBeatSfx();
+    }
+    
+    /// <summary>
+    /// The initiator of heart beat coroutines (should only start up and stop once per calll at max)
+    /// </summary>
+    public void PlayHeartBeatSfx()
+    {
+        if (_currentHealth <= _heartToStartHeartSfx && _currentHealth != _heartToEndHeartSfx && _heartBeatCoroutine == null)
+        {
+            StartCoroutine(HeartbeatLoop());
+        }
+        if (_currentHealth >= _heartToEndHeartSfx && _heartBeatCoroutine != null)
+        {
+            StopCoroutine(HeartbeatLoop());
+        }
+    }
+
+    /// <summary>
+    /// A coroutine that is mean't to loop the heartbeat sfx
+    /// </summary>
+    /// <returns>returns null to loop similar to update function</returns>
+    private IEnumerator HeartbeatLoop()
+    {
+        while (true)
+        {
+            yield return new WaitForSeconds(_heartBeatRateSfx);
```

Comment:
i think the wait should be after the sfx so it plays once then waits instead of delaying the first play